### DISCUSSION TITLE
What is the use of Global Parameters?

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#108] [General]
Hello everyone!
                        In the runScript.py file, we have some global parameters. Majorly, I am interested in optimizing an airfoil at a particular velocity and angle of attack. If I only change it in runScript file, would it be alright? or for running it at the desired conditions, I need to change somewhere else also??

Moreover, global parameters seem to be initial value kind of things. Does it mean that the values will change during the optimization process?
 
Kindly help me in this regard.
Thanks...

**Comment by friedenhe** [comment]
If you want to change the flight condition, you need to change the U0, p0, alpha0, etc in runScript.py that is it. During optimization, U0 will not change, but alpha (angle of attack) will change (if you set it as a design variable).
